dataclients/kubernetes/testdata/routegroups/traffic/issue2268.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  namespace: ns
+  name: issue2268
+spec:
+  hosts:
+    - test.example.org
+  backends:
+    - name: service-1
+      type: service
+      serviceName: service-1
+      servicePort: 8080
+    - name: service-2
+      type: service
+      serviceName: service-2
+      servicePort: 8080
+    - name: shunt
+      type: shunt
+  routes:
+    # custom route
```

Comment:
how do you define `custom route` in this context?
1. a non service route (by backend type for example)
2. a route that is not part of the traffic switching

Reply:
the 2.

Reply:
The route is defined by its predicates, filters and backends. "custom" is just a name.